pyproject.toml

```diff
@@ -112,8 +112,8 @@ pyhive = { version = "^0.7.0", extras = ["hive_pure_sasl"], optional = true }
 pymongo = "^4.6.3"
 pyopenssl = "^24.1.0"
 python = ">=3.9,<4"
-python-multipart = "^0.0.17"
 python-on-whales = ">=0.60.0"
+python-multipart = "*"
```

Comment:
Is it possible to specify some limit on the version to prevent unexpected issues caused by newer releases?

Reply:
I don't think its possible for python-multipart. They have are at version 0.0.XX. 

Makes more sense if the packages are at least 1.0

Reply:
I meant pinning to a specific version like `0.0.20` unless there is a reason to always fetch the latest version.

Reply:
Oh it's because fastapi-users is required in featurebyte-app but not in featurebyte. 
And that package pins python-multipart to 0.0.20.
So if I hard pin it in featurebyte, I might not be able to upgrade fastapi-users in featurebyte-app side unless I bump it in featurebyte first.